api-mocks/README.md

```diff
@@ -0,0 +1,31 @@
+# API Mocks
+
+CogBoard uses WireMock for testing. As corresponding functional tests are required for acceptance of new widget some of those tests will require mocks of services the widget uses.
```

Comment:
Maybe:
`CogBoard uses WireMock to mimic responses of third party software that can be a data source for Cogboard. This allows us to tests widgets without need to keep whole infrastructure of all third party software related to Cogboard.`

WDYT?

Reply:
Definitely looks nicer